ivy.el

```diff
@@ -3956,6 +3963,24 @@ characters (previous if ARG is negative)."
   (interactive "p")
   (ivy--yank-by #'forward-char arg))
 
+(defvar ivy--pulse-overlay nil
+  "Overlay used to highlight yanked word.")
+
+(defvar ivy--pulse-timer nil
+  "Timer used to dispose of ivy--pulse-overlays.")
+
+(defun ivy--pulse-region (begin end)
+  (unless ivy--pulse-overlay
+    (setq ivy--pulse-overlay (make-overlay 0 0)))
```

Comment:
Zero isn't a valid buffer position, so maybe use `(point)` instead?

Reply:
the overlay is not meant to be displayed anyway, but I can use min-point.